includes/service-providers/mailchimp/class-newspack-newsletters-mailchimp-cached-data.php

```diff
@@ -300,11 +307,19 @@ private static function clear_errors( $list_id ) {
 	/**
 	 * Stores the last error for a given list, if the cache is older than self::SURFACE_ERRORS_AFTER
 	 *
-	 * @param string $list_id The List ID.
-	 * @param string $error The error message.
+	 * @param string|null $list_id The List ID, or null for all lists.
+	 * @param string      $error The error message.
 	 */
-	private static function maybe_add_error( $list_id, $error ) {
-		Newspack_Newsletters_Logger::log( 'Mailchimp cache: handling error while fetching cache for list ' . $list_id );
+	private static function maybe_add_error( $list_id = null, $error ) {
```

Comment:
Having an optional parameter before a required paramater throws a PHP deprecated error
```suggestion
	private static function maybe_add_error( $list_id, $error ) {
```

Reply:
Ah, `$list_id` needs to be optional because it's `null` if we're adding an error about refreshing the cache for audience data. 017efcc should fix it by making `$error` also optional.